tron/api/resource.py

```diff
@@ -350,13 +357,45 @@ def render_GET(self, request):
             "include_node_pool",
             default=True,
         )
+
+        page = requestargs.get_integer(request, "page")
+        page_size = requestargs.get_integer(request, "page_size")
+
+        jobs = self.get_data(
+            include_job_runs,
+            include_action_runs,
+            include_action_graph,
+            include_node_pool,
+            page,
+            page_size,
```

Comment:
we're using page/page_size here before validating them - is that intentional?

Reply:
Great catch!!! this is a good point!!! most of the page validation happens a bit after on the code, handling exceptions here makes more sense.